Plugins/Bitbucket/BitbucketPullRequestForm.cs

```diff
@@ -30,6 +29,10 @@ public BitbucketPullRequestForm(BitbucketPlugin plugin, ISettingsSource settings
             InitializeComponent();
             Translate();
 
+            // NOTE ddlBranchSource and ddlBranchTarget both have string items so do not need a display member
```

Comment:
Thanks for comments like this

Reply:
So if they don't need this, why do have it?

Reply:
I believe this is a preparation for private BitBucket repositories, almost implemented. I do not have access to a setup like that and preserved the existing implementation.

Reply:
Ahhh... indeed I have. Apologies